homeassistant/components/plugwise/strings.json

```diff
@@ -20,11 +20,12 @@
       },
       "user_gateway": {
         "title": "Connect to the Smile",
-        "description": "Please enter:",
+        "description": "Please enter",
         "data": {
           "password": "Smile ID",
           "host": "[%key:common::config_flow::data::ip%]",
-          "port": "[%key:common::config_flow::data::port%]"
+          "port": "[%key:common::config_flow::data::port%]",
+          "username" : "Smile Username"
```

Comment:
this should use common string for username like password does

Reply:
The username and password are predefined values, they are preset for each Smile gateway by the manufacturer.
The names "Smile Username" and "Smile ID" match the naming used by the manufacturer. 
Therefore we would like to specifically keep them.